build_mac.sh

```diff
@@ -7,6 +7,7 @@ sed s/\$\{arch\}/x86_64/g < pom1.xml  > pom.xml
 mvn -Dos=macosx -Darch=x86_64 clean install
 cp pom_template.xml pom.xml
 rm pom1.xml
+rm pom_template.xml
```

Comment:
Maybe I'm missing some part in the build process as to why this file isn't cleaned up in this script, but the result of not deleting it is that it leaves a file that Git then detects as a change, leaving my working state dirty even if I made no changes.